### ISSUE TITLE
[DOXIA-294] Apt verbatim box not correct

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-doxia#930]
**[Jakob Vad Nielsen](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=lazee)** opened **[DOXIA-294](https://issues.apache.org/jira/browse/DOXIA-294?redirect=false)** and commented

It seems to me that there are a mess on what Doxia packages versions that are chosen when rendering a site with mvn site:site. See the attached dump.txt. It seems to me that version 1.0 of doxia is not chosen. And that older versions are used.

I have added this to my pom.xml

plugin>
\<groupId>org.apache.maven.doxia\</groupId>
\<artifactId>doxia-maven-plugin\</artifactId>
\<version>1.0\</version>
\</plugin>

I discovered this when trying to find out why verbatime text doesn't work like it did before from an apt file.

---

Verbatime

---

gives thes same markup as

+----------------------+
Verbatime 2
+----------------------+

Both are converted into \<div class="source">\<pre>Verbatime\</pre>\</div>. This is a new bug that I haven't seen before.

I have attached a small maven project that demonstrates this. demo.html generated with site:site demonstrates the problem.

---

**Affects:** 1.1

**Attachments:**
- [dump.txt](https://issues.apache.org/jira/secure/attachment/12715482/dump.txt) (_107.86 kB_)
- [proof.zip](https://issues.apache.org/jira/secure/attachment/12715419/proof.zip) (_144 bytes_)

**Issue Links:**
- [DOXIA-297](https://issues.apache.org/jira/browse/DOXIA-297) Verbatim blocks are always boxed

**Comment by jira-importer** [comment]
**[Jakob Vad Nielsen](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=lazee)** commented

The second verbatime lines came out the wrong way. Should have been a single line with plus in front and rear.

+ - - - - - - - + (without the spaces)

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

I don't see anything unusual in dump.txt, what exactly are you referring to? See also MNG-3402, (maven always uses the embedded doxia version).

The attached zip is empty, can you update it?

**Comment by jira-importer** [comment]
**[Jakob Vad Nielsen](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=lazee)** commented

It might be me that do not understand the dependencies correctly. But why are older versions of Doxia used?

e.g.:

[DEBUG] Retrieving parent-POM: org.apache.maven:maven-parent:pom:6 for project: org.apache.maven.doxia:doxia:pom:1.0-alpha-10 from the repository.
[DEBUG] Adding managed dependencies for unknown:doxia-core
[DEBUG]   org.apache.maven.doxia:doxia-sink-api:jar:1.0-alpha-10
[DEBUG]   org.apache.maven.doxia:doxia-core:jar:1.0-alpha-10
[DEBUG]   org.apache.maven.doxia:doxia-core:jar:tests:1.0-alpha-10

Or am I reading it wrong?

But the important issue for me here is that I do not get different markup for ----------- ad + ----------- + anymore. Up until last week I was able to separate these verbatims from eachother via my special maven theme. Now it gives me the exact same output.

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

The different versions are the versions used by various transitive dependencies, they are not all used actually in the final classpath.

I confirm the verbatim box issue, we'll fix it after the 1.1 release.

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

Fixed in r751515

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

I just realized that we have the same problem in 1.0 too. It's a different issue though, for 1.1 the bug was in AptParser (this one I fixed), for 1.0 it's in XhtmlSink. The effect is the same, verbatim blocks are always boxed. I'm not sure if we'll do a 1.0.1 release but I'll open another issue for it.
